hack/build-image

```diff
@@ -219,6 +231,11 @@ def container_build(cli, target):
         # --arch is not provided. So if the target arch and the host_arch
         # are the same, skip passing the extra argument.
         args.append(f"--arch={target.arch}")
+
+    pkgs_from = PACKAGES_FROM[target.pkg_source]
+    if pkgs_from:
+        args.append(f"--build-arg=INSTALL_PACKAGES_FROM={pkgs_from}")
+    
```

Comment:
I'm not clear on what this addition is doing. Can you elaborate?

Reply:
I did not add those lines; I only moved this block down a little bit.

Reply:
Yes, I missed that, thank you.